### ISSUE TITLE
Enable secret output for rhoas generate-config

### ISSUE THREAD

**Issue by wtrocki** [redhat-developer/app-services-cli#1535]
## Description

Config generation should produce kuberentes secret. Secret should use env var type keys that will allow developers to mount secret as environment variables.

**Comment by wtrocki** [comment]
Open Question - Should we add the same for service account create command as well?

**Comment by wtrocki** [comment]
```
apiVersion: v1
kind: Secret
metadata:
  name: mysecret
type: Opaque
stringData:
  RHOAS_SERVICE_ACCOUNT_CLIENT_ID: YWRtaW4=
  RHOAS_SERVICE_ACCOUNT_SECRET: MWYyZDFlMmU2N2Rm
 ```

**Comment by rkpattnaik780** [comment]
Closed by #1539
